server/controllers/messages.coffee

```diff
@@ -194,12 +196,30 @@ module.exports.send = (req, res, next) ->
     delete req.body.isDraft
 
     message = req.body
-    message.html = message.html.trim() if message.composeInHTML
+    if message.html
+        minifierOpts =
```

Comment:
Ok, I am a morron, though this was on client-side...
Request still applies but for code readability.